src/app/modules/shared.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NgModule } from '@angular/core';
+import { LoginService } from '../services/login.service';
+import { LoginComponent } from '../pages/login';
+import { FormsModule } from '@angular/forms';
+import {
+  RouterModule,
+  PreloadAllModules
+} from '@angular/router';
+
+@NgModule({
+  declarations: [
+    LoginComponent,
```

Comment:
jeśli postanowiłeś zrobić shared module to może warto też dodać wszystkie współdzielone komponenty np header, footer + serwisy